python/ray/serve/tests/test_api.py

```diff
@@ -884,6 +885,105 @@ async def __call__(self, request: Request) -> Dict[str, str]:
     }
 
 
+def test_status_basic(serve_instance):
+    # Before Serve is started, serve.status() should have an empty list of applications
```

Comment:
Could you also assert that the `http_proxies` are empty?

Reply:
Actually since this uses the shared serve instance, Serve has already been started so the list of http proxies is non empty.

Reply:
Oh got it, I'm okay with leaving this as-is then.